lib/libutee/tee_system_pta.c

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2019, Linaro Limited
+ */
+
+#include <pta_system.h>
+#include <string.h>
+#include <tee_internal_api_extensions.h>
+#include <tee_internal_api.h>
+#include <types_ext.h>
+#include <util.h>
+
+static TEE_Result invoke_system_pta(uint32_t cmd_id, uint32_t param_types,
+				    TEE_Param params[TEE_NUM_PARAMS])
+{
+	static TEE_TASessionHandle sess = TEE_HANDLE_NULL;
+	static const TEE_UUID uuid = PTA_SYSTEM_UUID;
+
+	if (sess == TEE_HANDLE_NULL) {
+		TEE_Result res = TEE_OpenTASession(&uuid, 0, 0, NULL,
+						   &sess, NULL);
+
+		if (res)
+			return res;
+	}
+
+	return TEE_InvokeTACommand(sess, 0, cmd_id, param_types, params, NULL);
+}
+
+void *tee_map_zi(size_t len, uint32_t flags)
+{
+	uint32_t param_types = TEE_PARAM_TYPES(TEE_PARAM_TYPE_VALUE_INPUT,
+					       TEE_PARAM_TYPE_VALUE_INOUT,
+					       TEE_PARAM_TYPE_VALUE_INPUT,
+					       TEE_PARAM_TYPE_NONE);
+	TEE_Param params[TEE_NUM_PARAMS];
+	TEE_Result res = TEE_SUCCESS;
+
+	memset(params, 0, sizeof(params));
```

Comment:
s/memset()/TEE_MemFill()/?
Or remove since initialized where defined.

Reply:
I'll remove it.